src/services/ldap/index.js

```diff
@@ -45,6 +46,27 @@ module.exports = function LDAPService() {
 				});
 		}
 
+		/** Used for activation only */
```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5713943)